### PR TITLE
Multiprotocol listing updates for 2.2

### PR DESCRIPTION
**Pull request by MRC3742**
Multiprotocol listing updates for both Radio UI and Companion.
Changes, updates to bring OpenTX Multi listings in line to current Multiprotocol Module version 1.2.1.0

Compiled, tested and working using v2.2.3 on X9D+

This PR does not address problems with saving "Custom" settings into Companion and no "Custom" option in Companion listed in issue #5679

### CODE REVIEW

**Comment by LapinFou** [comment]
Since the development of the Multi project will never be align with the timeline of the OpenTX development, I got a suggestion:
Would it be possible to store the protocol information on the SD card?
Thus, updating the multi module with new protocols will not require a new OpenTX version. Only updating a text file will be require.

**Comment by schwabe** [comment]
Yes would be possible but I never got around writing the code. And if I implement something like that, the module should send the protocol information and not the SD card

**Comment by 3djc** [comment]
> the module should send the protocol information and not the SD card

Indeed !

**Comment by LapinFou** [comment]
Fair point.
I will post an enhancement request on Pascal's GitHub.

**Comment by LapinFou** [comment]
After a 2nd thought, maybe it would be difficult to do it now without breaking the ascendant compatibility...

Here is the ticket I opened → https://github.com/pascallanger/DIY-Multiprotocol-TX-Module/issues/193

**Comment by LapinFou** [comment]
Some news on https://github.com/pascallanger/DIY-Multiprotocol-TX-Module/issues/193
Looks like Pascal is not against the idea. 😄

**Comment by schwabe** [comment]
@LapinFou Yeah. But I have currently not much time for OpenTx development. The Multimodule side is the easy one.

**Comment by LapinFou** [comment]
Well, as long as everyone like the idea, there is some good chance that something will be done one day.
That is the most important. 😄

**Review by schwabe — APPROVED** [review]

**Comment by MRC3742** [comment]
Thank You for the approval as I was confident this was complete and tested when PR was made.

For anyone with a better understanding of commits,
Is there a way to change multiple files using one commit ?
This PR looks like a lot of commits but each translation/*.txt file was just a copy, paste, commit.
Using the web interface I had to commit on every file change or file update.
I believe that these can still be squashed into one commit before it is merged by maintainers ?
Thanks - Rich

**Comment by kilrah** [comment]
You cannot edit multiple files using the web interface, you need to clone locally and use the "normal" git workflow.
We will squash, no issue.

**Comment by MRC3742** [comment]
Thanks - I'll try that approach in the future.

Does another PR need to be submitted for this to be added to the 2.3 branch? or possibly are you hoping to have that version updated from the MPM via serial as is being suggested.

File: radio/src/translations/de.h.txt
Comment (bsongis) [inline radio/src/translations/de.h.txt:118]: I wonder if we really need to translate the protocols list, are there any differences? If no I propose to move them in translations.h

**Comment by bsongis** [comment]
Needed in 2.3

File: radio/src/translations/de.h.txt
Comment (schwabe) [reply radio/src/translations/de.h.txt:118]: No probably not.

**Pull request merged**
